test/Chainweb/Test/Pact/PactExec.hs

```diff
@@ -65,7 +66,7 @@ pactTestSetup :: IO PactTestSetup
 pactTestSetup = do
     let loggers = alwaysLog
     let logger = newLogger loggers $ LogName "PactService"
-    let pactCfg = pactDbConfig (Test petersonChainGraph)
+    let pactCfg = pactDbConfig testVersion
```

Comment:
`pactExecTests` below uses `Testnet00` as chainweb version. I assume that mixing different chainweb version in throughout these tests was not intended.

Reply:
Correct